### ISSUE TITLE
Nuxt VS Lighthouse v6 a fast site with poor lighthouse performance score

### ISSUE THREAD

**Issue by balazs9703** [GoogleChrome/lighthouse#10857]
### Provide the steps to reproduce
Run LH on [https://lighthouse-test.blackjuice.eu/](https://lighthouse-test.blackjuice.eu/)
Run V6 with npm package from CLI on [https://lighthouse-test.blackjuice.eu/](https://lighthouse-test.blackjuice.eu/)

### What is the current behavior?
We run the lighthouse v6 and V5 on our nuxt.js application and it show low performance but we tested on multiple test sites with different settings but we couldn't get close to LH score. We always get better resultes.
Devtools lighthouse shows 90+ score.

### What is the expected behavior?
Show better performance score that is closer to other measurements and real life performance.

### Environment Information
Lighhouse version: web.dev, 6.0
Nuxt: v2.12.2
Node: v10.20.1
Operating System: Ubuntu 19.10

### Tests:
[https://ibb.co/NLmwbf9](https://ibb.co/NLmwbf9) Devtool LH
[https://ibb.co/Gt3byBy](https://ibb.co/Gt3byBy) GTMetrix
[https://ibb.co/n3WrB1Q](https://ibb.co/n3WrB1Q) web.dev
[https://ibb.co/WkKTywz](https://ibb.co/WkKTywz) LH v6 from CLI
[https://ibb.co/d79q9TW](https://ibb.co/d79q9TW) Webpagestest.org

**Comment by patrickhulce** [comment]
Thanks for filing @balazs9703!

Everything I'm seeing for that URL is working as intended. It's worth noting I do see a high degree of variability for that page (see [variability docs](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) for more), but all scores I got were in the ~60-50 range on v6 and ~70-60 range on v5. The page doesn't do very well on the new Largest Contentful Paint and Total Blocking Time metrics, so I would expect a bit of score drop.

As for the specific tests you've referenced...

> https://ibb.co/NLmwbf9 Devtool LH

It looks like the page did not load properly. Notice how the screenshots at the bottom are very slim rectangles. The stats also suggest this was unlike all other page loads if content painted in basically the theoretical min at 0.5s. The runtime stats are cutoff in the screenshot but I would guess they're not the same environment either.

> https://ibb.co/Gt3byBy GTMetrix

GTMetrix is not measuring, to use your words, "real life performance". Those checks you screenshoted are a checklist of best practices but do not in any way guarantee specific performance. 

> https://ibb.co/n3WrB1Q web.dev
https://ibb.co/WkKTywz LH v6 from CLI

Both of these are in line with what I saw in the ~50s. WAI.

> https://ibb.co/d79q9TW Webpagestest.org

I can't see what was going on with this one because the Lighthouse report isn't in it, but I was unable to replicate the performant numbers there. In fact, I couldn't get Lighthouse to successfully complete on that URL in WPT, a separate issue but still.

**Comment by connorjclark** [comment]
meta note: FYI in GitHub you can just drag and drop images to your PR description, no need to use an external image host :)

**Comment by patrickhulce** [comment]
I was finally able to run the URL on WPT and got a 16, so even worse metrics on a real device https://webpagetest.org/lighthouse.php?test=200527_85_7eaec2b4be9ad59b20e1432510ba935a&run=1

Probably more [variability](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) to blame with how bad the score is, but definitely not being overly pessimistic with a score of 50.

**Comment by balazs9703** [comment]
@patrickhulce 
I rerun the devtool LH test with proper size:
![image](https://user-images.githubusercontent.com/52198370/83064254-ca07c480-a061-11ea-90e1-caffdcba7e3e.png)

Webpagetest.org test: [https://webpagetest.org/result/200527_CM_a5ac2fa016a25d35d013d31fa52df21a/](https://webpagetest.org/result/200527_CM_a5ac2fa016a25d35d013d31fa52df21a/)

Problem is that on webpagetest.org and on devtool LH I got ~90+ score but on web.dev and cli LH v6 ~40-60.
We tested the website on a lot of devices with different network speed, without cache, with throttled network and cpu but never got this bad load time.

TL;DR: We cannot reproduce in any shape of form that slow load times with real devices or even simulated device with network and CPU throttle. We got 16-97 performance points on different type of LH. Which is a huge spread.

Why we got this huge spead? Do you see any problem with the site that can cause this?
We use nginx with gzip, nuxt.js with Vuex, PWA. We use it as jamstack so it is fully static.

**Comment by patrickhulce** [comment]
Is this site hosted somewhere close to Frankfurt, Germany?

For web.dev specifically, I'm guessing it's the fact that web.dev/measure will test from US central compared to Frankfurt. Every test on webpagetest.org I've run from Dulles, VA, United States gets much worse results than the tests you've linked which all run in Frankfurt. Factors like location discussed in the [variability docs](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) can have an enormous effect which is *expected* behavior. If you don't have a server near the test location, and a user tries to visit from there, the experience will be poor. Do you see the same results for[ PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/) as you do for web.dev/measure? 

For the CLI...

- Where are you running the CLI from in relation to the server? 
- Would you be able to share traces (the files created when running with `--save-assets`)?
- Is there a large difference with `--throttling-method=devtools`?

**Comment by balazs9703** [comment]
Do you see the same results for PageSpeed Insights as you do for web.dev/measure?
I got very similar results. ~70.

Where are you running the CLI from in relation to the server?
Server is in Germany and I'm testing from Hungary, Budapest.

Would you be able to share traces (the files created when running with --save-assets)?
You can download from here. https://www.dropbox.com/s/n5dqcz5ijtfqi4w/lighthouse-test.blackjuice.eu_2020-05-28_00-15-50.report.zip?dl=0

Is there a large difference with --throttling-method=devtools?
Tested multiple times and It shows about 10 point difference. 
With --throttling-method=devtools I got 10 point less.

**Comment by patrickhulce** [comment]
Thanks for sharing the trace! I see nothing unexpected in that trace that needs fixing in Lighthouse though. The metrics are legitimately that large there.

DevTools and environments without Lighthouse have different characteristics from that will tend to make results different (you can disable storage clearing, it's using a warm cache when that happens, other Chrome profile factors and DevTools settings affect results, etc). The fact that FCP is 0.5s there is an immediate hint the environment is not comparable because that's next to impossible on Slow 4G https cold load. There's nothing actionable for us here as everything seems explained by the factors described in [our documentation](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md). Sorry we couldn't be of more help :/
